plugins/org.eclipse.fordiac.ide.typemanagement/src/org/eclipse/fordiac/ide/typemanagement/refactoring/InsertMuxCommand.java

```diff
@@ -0,0 +1,16 @@
+package org.eclipse.fordiac.ide.typemanagement.refactoring;
+
+import org.eclipse.fordiac.ide.model.libraryElement.Connection;
+import org.eclipse.gef.commands.Command;
+
+public class InsertMuxCommand extends Command {
```

Comment:
please rename Insert StructManipulart as this is the common base class for muxer and demuxer